docs/k8s-helm.md

```diff
@@ -75,11 +71,10 @@ timeplusd:
       memory: 4Gi
 ```
 Then make changes to better fit your need.
-1. Update the storage class name and size accordingly. You can check available storage classes on your cluster by running `kubectl get storageclass`. Common values are `local`, `standard`, `ebs-gp3-basic-auto-delete`, `ebs-gp3-basic-encrypted`, etc.
```

Comment:
> Common values are

There is no such concept of `Common values` for storage class. Or put it into another word, whether it is a `Common values` or not does NOT impact the user's decision here. 

Besides, I believe`ebs-gp3-basic-auto-delete` and `ebs-gp3-basic-encrypted` are defined by ourselves. The user won't be able to find them even if they are using EKS

Reply:
We dont need to mention `ebs-gp3-basic-auto-delete`, `ebs-gp3-basic-encrypted` But still need to highlight the user need to change this storage class. Maybe on AWS, also mention the recommended IOPS for EBS, say gp3, iot2 (is that 3000 as our recommendation?)

Reply:
I've added `Please check the [Planning capacity](#planning-capacity) section for storage recommendations` to this section.